openhands-cli/pyproject.toml

```diff
@@ -98,7 +98,3 @@ warn_return_any = true
 warn_unused_configs = true
 disallow_untyped_defs = true
 ignore_missing_imports = true
-
-[tool.uv.sources]
```

Comment:
Do we still want this? I believe it only applies to local development and is a way to test unreleased agent-sdk code

Would be good to double check this though